src/util/windows.jl

```diff
@@ -0,0 +1,66 @@
+"""
+An iterator which moves across a container, as a sliding window.
+"""
+immutable EachWindowIterator{T <: AbstractArray}
```

Comment:
This type parameter constraint is too strict to apply the window iterator to biological sequences.